test/test_serial_port/test_serial_port.cpp

```diff
@@ -62,7 +62,7 @@ void test_subscribeToLine()
         When(Method(ArduinoFake(Serial), read)).Return(testLine[i]);
         serialEvent();
     }
-    TEST_ASSERT_EQUAL_STRING(testLine, receivedLine.c_str());
+    TEST_ASSERT_EQUAL_STRING("C++", receivedLine.c_str());
```

Comment:
was this change intentional?
I just found this, as I tried to find a solution for the conflict in the platformio.yml.

Reply:
Yes this change was intentional: The algorithm for reading lines has changed as such that the end-of-line characters are now discarded. That is why `testLine = "C++\n"` can not be used for comparison any more.